db/migrate/20231107210703_create_join_table_dish_ingredient.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableDishIngredient < ActiveRecord::Migration[7.0]
+  def change
+    create_join_table :dishes, :ingredients do |t|
+      # t.index [:dish_id, :ingredient_id]
```

Comment:
Another rails migration command you could use to create a joins table:
`rails g migration CreateDishIngredients dish:references ingredient:references`

this will add the foreign keys in the joins table

Reply:
Rather than needing to do it in a separate migration